lib/utils.js

```diff
@@ -81,9 +128,8 @@ function mergeQuery(base, update, spec) {
     if (!base.include) {
       base.include = update.include;
     } else {
-      var saved = base.include;
-      base.include = {};
-      base.include[update.include] = saved;
+      //merge inclusion
+      base.include = mergeIncludes(base.include, update.include);
```

Comment:
@raymondfeng, Latest push has fixed the failed test case.
https://travis-ci.org/walkonsocial/loopback-datasource-juggler/jobs/59677367